TrackerEvents.cs

```diff
@@ -91,32 +68,22 @@ internal static void RegisterRunEventHandlers(DynamoModel model)
         /// <param name="e"></param>
         private static void OnEvaulationCompleted(object sender, EvaluationCompletedEventArgs e)
         {
-            /*
-            if (FileName == null)
-            {
-                FileName = "New Graph";
-            }
-            */
-
-            // @todo if EvaluationCount greater than 1..
+            // Display a MessageBox to the user
+            // @todo Remove this as it's for testing purposes only
+            string filename = ExportData.filename;
+            MessageBox.Show($"The current Graph name is {filename}");
 
-            // WORKS
-            MessageBox.Show($"The current Graph name is {FileName}");
-
-            var dataToExport = ExportData.Export(FileName, DynamoVersion);
-            ExportSheets.Execute(dataToExport);            
+            // When graph evaluation is complete
+            // Get all the data we want to store and then store it
+            var dataToExport = ExportData.Export();
+            ExportSheets.Execute(dataToExport);
         }
 
         private static void OnGraphRun(object sender, bool success)
         {
             // DOES NOT WORK
             if (success)
             {
-                string DynamoVersion = string.Empty;
-                MessageBox.Show($"The current Graph name is {FileName}");
-
-                var dataToExport = ExportData.Export(FileName, DynamoVersion);
-                ExportSheets.Execute(dataToExport);
             }
```

Comment:
why does this not work ? I'd either
- trace the issue & fix
- leave it as is
- delete the entire contents of the methods, leaving the `if` in there doesn't serve any purpose

Reply:
Because of my commits I think the line reference above has changed and should refer to:

https://github.com/teamtreedyn/Binoculars/blob/master/Events.cs#L37
```c#
        private static void OnGraphRun(object sender, bool success)
        {
            // DOES NOT WORK
            if (success)
            {
            }
        }
```

The answers to this is Dynamo's `RunCompleted` event is simply never called by Dynamo:  https://github.com/DynamoDS/Dynamo/issues/9602

Therefore, we will remove this event and continue to use `EvaluationCompleted`

Reply:
Fixed in https://github.com/teamtreedyn/Binoculars/commit/5df70272967bfc013f6b71a0cd51f19e9165f0b0 therefore marking resolved.